docs/website/11ty/articles.filter.js

```diff
@@ -1,12 +1,12 @@
 /** Filters to extract article (rich text) parts */
-module.exports = (config) => {
+export default (config) => {
   config.addFilter('extractArticleContent', (text) => {
     const contentRegex = /<p.*>.*?<\/p>/ig;
-    return text.match(contentRegex).join(' ');
+    return text && text.match(contentRegex) ? text.match(contentRegex).join(' ') : "";
```

Comment:
```suggestion
    return text && text.match(contentRegex) ? text.match(contentRegex).join(' ') : '';
```

Actually, the liter should be fixed to display the style inconsistency.
As I marked for package.json file the current lint:js script currently doesn't have correct mask